src/js/script.js

```diff
@@ -427,10 +430,16 @@
       thisCart.dom.totalNumber.innerHTML = thisCart.totalNumber;
       thisCart.dom.deliveryFee.innerHTML = thisCart.deliveryFee;
       thisCart.dom.subtotalPrice.innerHTML = thisCart.subtotalPrice;
+    }
 
-      for(let totalPriceHolder of thisCart.dom.totalPrice) {
-        totalPriceHolder.innerHTML = thisCart.totalPrice;
-      }
+    remove (){
+      const thisCart = this;
+      cartProduct.dom.wrapper.remove();
+
+      const indexOfProduct = thisCart.products.indexOf(cartProduct);
+      thisCart.products.splice(indexOfProduct, 1);
+
+      thisCart.update();
     }
```

Comment:
1. Tutaj powinnaś przekazać `cartProduct`. Dzięki temu będziesz mogła go usunąć z DOM wywołując metodę `remove` na wrapperze: cartProduct.dom.wrapper.remove()`.

Kod powinien wyglądać jak poniżej.
```suggestion
   remove(cartProduct){
      const thisCart = this;


      cartProduct.dom.wrapper.remove()
      const indexOfProduct = thisCart.products.indexOf(cartProduct);
      thisCart.products.splice(indexOfProduct, 1);

      thisCart.update();
    }
```